src/analysis/processing/qgsalgorithmalignrasters.cpp

```diff
@@ -116,7 +116,7 @@ QVariantMap QgsAlignRastersAlgorithm::processAlgorithm( const QVariantMap &param
   if ( parameters.value( QStringLiteral( "CRS" ) ).isValid() )
   {
     QgsCoordinateReferenceSystem crs = parameterAsCrs( parameters, QStringLiteral( "CRS" ), context );
-    customCRSWkt = crs.toWkt( QgsCoordinateReferenceSystem::WKT_PREFERRED_GDAL );
+    customCRSWkt = crs.toWkt( Qgis::CrsWktVariant::PreferredGDAL );
```

Comment:
Should it not be PreferredGdal? E.g., we have QgsGdalUtils, which does not type gdal in all-caps.

Reply:
Done